pom.xml

```diff
@@ -49,7 +49,9 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
-        <asciidoctorj.version>1.5.3.2</asciidoctorj.version>
+        <asciidoctorj.version>1.6.0-alpha.3</asciidoctorj.version>
```

Comment:
You refer to an exception when generating the PDF of HV but I don't see the exception anywhere? What's the issue exactly?